DOCS/man/options.rst

```diff
@@ -2786,13 +2786,23 @@ Subtitles
 
     Default: sdr
 
+    This option is disabled when ``--blend-subtitles=no`` and ``--target-colorspace-hint=no``
+    are both used. It is not guaranteed to work if ``--blend-subtitles=no`` and
+    ``--target-colorspace-hint=auto`` are both used (see ``--target-colorspace-hint``
+    for details).
+
 ``--sub-hdr-peak=<sdr|10-10000>``
     Controls the text subtitle and OSD diffuse white level in cd/m² (nits)
     for HDR output. ``sdr`` is 203 cd/m² for standard SDR white.
     (``--vo=gpu-next`` only)
 
     Default: sdr
 
+    This option is disabled when ``--blend-subtitles=no`` and ``--target-colorspace-hint=no``
+    are both used. It is not guaranteed to work if ``--blend-subtitles=no`` and
+    ``--target-colorspace-hint=auto`` are both used (see ``--target-colorspace-hint``
+    for details).
```

Comment:
`target-colorspace-hint` is not really relevant here, it's all encapsulated in `for HDR output`. How you enable HDR ouptut is another question. You can do `--target-trc=pq` and it would also work.


blend-subtitles is needed for color managing subtitles. There is a not already about it, but we can mention that it's needed, if it really is, because I'm not sure @dyphire agrees on this.

Reply:
> I may misunderstand the implement if it is only designed for HDR outout.

Yes, as the option describes it is designed for HDR output.
It's still possible to make it work in HDR tone mapping with `blend-subtitles=yes/video`, which is what I was hoping to add. This was missed in the documentation for the pr that implements it.